### PR TITLE
Team $we init, introductory post.

### PR DESCRIPTION
**Pull request by Rahmeen14**

### CODE REVIEW

File: blog/_posts/2018-07-09-$we-init-introductory-post.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Dreams come true !
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-$we-init-introductory-post.md]: Hi @Rahmeen14 ! Thanks for the PR. Please check the other blog posts for the proper markdown format. We need the whole set of parameters at the start.

File: blog/_posts/2018-07-09-$we-init-introductory-post.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+### Mananpal (Coach)
+A senior year student from our college, Mananpalhas won almost every hackathon he has ever participated it. Being an extremely passionate developer and open-source enthusiast, Manan has this knack for developing some really cool projects. He works actively for promoting open-source among the students of our college and is an amazing person.
+
+### Saptak (Supervisor)
+A graduate from IIIT Allahabad, Saptak is a passioante open source contributor. A GSoC '16 student with FOSSASIA, Saptak is currently working as a Software Developer with Gitcoin. We're super excited to work with him throughout the summer!  
+
+### Say Cheese! 
+
+![Team $we init in one frame](/img/blog/2018/WeInit.jpg)
+
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-$we-init-introductory-post.md]: Please add image description and use image-credits class to add credits.

File: blog/_posts/2018-07-09-$we-init-introductory-post.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+Heya! We are Rahmeen and Akshita of team $we init. We both study in New Delhi, India, and are working on Probot this summer. 
+
+## The Dreamers:
+
+### Rahmeen 
+* She is incredibly ticklish and smiles and laughs hysterically in ways that are most contagious.
+* She is super clumsy and is capable of tripping over absolutely nothing
+* She writes a lot to mute the demons in her head and reads voraciously. You can find her scrolling through her Quora feed almost always.
+* She has this habit of quoting random Shakespearean, Tagore and Frost quotations and then justifying their relevence in the ongoing context.
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-$we-init-introductory-post.md]: Fix the typo here: 'relevance'.

File: blog/_posts/2018-07-09-$we-init-introductory-post.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+### Manish (Coach)
+Our bachmate and a 2018 GSocer, Manish is a very good friend of ours. Being a CodeHeat winner under FOSSASIA, he is extremely passionate about contributing to open source and loves to help others too. 
+
+### Mananpal (Coach)
+A senior year student from our college, Mananpalhas won almost every hackathon he has ever participated it. Being an extremely passionate developer and open-source enthusiast, Manan has this knack for developing some really cool projects. He works actively for promoting open-source among the students of our college and is an amazing person.
+
+### Saptak (Supervisor)
+A graduate from IIIT Allahabad, Saptak is a passioante open source contributor. A GSoC '16 student with FOSSASIA, Saptak is currently working as a Software Developer with Gitcoin. We're super excited to work with him throughout the summer!
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-$we-init-introductory-post.md]: Fix the typo here: 'passionate'

File: blog/_posts/2018-07-09-$we-init-introductory-post.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+### Saksham (Coach)
+A two-time GSoCer a 2017 passout from Delhi Technological University, Saksham is always happy to help. And not most people will believe but we found him on Quora (Yay, Rahmeen's infinite quora scrolling is not always useless). Saksham has a lot of experinece with Node.js and loves to share his knowledge abundantly. Workaholic as he is, he's worried about lack of work when we don't ask him doubts too frequently, and very cleverly this his how he keeps us on our toes. (Don't worry we'll trouble you a lot in the days to come XD)
+
+### Manish (Coach)
+Our bachmate and a 2018 GSocer, Manish is a very good friend of ours. Being a CodeHeat winner under FOSSASIA, he is extremely passionate about contributing to open source and loves to help others too.
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-$we-init-introductory-post.md]: Fix the typo here: 'batchmate'

File: blog/_posts/2018-07-09-$we-init-introductory-post.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+### Divjot (Coach)
+A 2016 batch passout from our college, Divjot is a full-time Software Developer at Swiggy. We wouldn't be exaggerating if we say that he's the absolute best senior one could ask for. From resolving out most messed up javascript doubts in minutes to giving detailed life lessons (and one liners like "Don't worry! If you want to give up, you're doing something right"), he is one person we can always depend on, without a second thought. He's vegan and Pun-jabi, funny and a hard taskmaster, loves JS and cooks amazing food (You've got to check his insta. @bogas04, thank us later for making you famous overnight :D)
+
+### Saksham (Coach)
+A two-time GSoCer a 2017 passout from Delhi Technological University, Saksham is always happy to help. And not most people will believe but we found him on Quora (Yay, Rahmeen's infinite quora scrolling is not always useless). Saksham has a lot of experinece with Node.js and loves to share his knowledge abundantly. Workaholic as he is, he's worried about lack of work when we don't ask him doubts too frequently, and very cleverly this his how he keeps us on our toes. (Don't worry we'll trouble you a lot in the days to come XD)
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-$we-init-introductory-post.md]: Fix the typo here: 'experience'

File: blog/_posts/2018-07-09-$we-init-introductory-post.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+### Bex and Jason (Co-mentors)
+They are both GitHub employees and maintainers of probot for quite some time now. They have been amazing mentors and have always offered helped, generously. :')
+
+### Divjot (Coach)
+A 2016 batch passout from our college, Divjot is a full-time Software Developer at Swiggy. We wouldn't be exaggerating if we say that he's the absolute best senior one could ask for. From resolving out most messed up javascript doubts in minutes to giving detailed life lessons (and one liners like "Don't worry! If you want to give up, you're doing something right"), he is one person we can always depend on, without a second thought. He's vegan and Pun-jabi, funny and a hard taskmaster, loves JS and cooks amazing food (You've got to check his insta. @bogas04, thank us later for making you famous overnight :D)
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-$we-init-introductory-post.md]: Change insta -> instagram

File: blog/_posts/2018-07-09-$we-init-introductory-post.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+To be able to work on a project of such reach and impact under the guidance of brilliant developers from across the world drew us towards Probot and the humility, support and niceness of these amazing developers has held us in it!
+
+## The Support System 
+
+### Gregor (Mentor)
+A member of the RGSoC family for a fourth year now, Gregor is one gem of a mentor. We cannot forget how warmly he welcomed us to the community, inviting us for office hours right on day 1 of introduction. Since then, he has been a constant support, has always kept us motivated and has made us feel so comfortable that we feel that we belong with Probot already! <3 
+
+### Bex and Jason (Co-mentors)
+They are both GitHub employees and maintainers of probot for quite some time now. They have been amazing mentors and have always offered helped, generously. :')
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-$we-init-introductory-post.md]: Change 'offered helped -> offered help.

File: blog/_posts/2018-07-09-$we-init-introductory-post.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+
+## Why Probot?
+Having worked with Javascript for almost a year now, we have fallen in deep love with it. NodeJS is one framework that we are extremely passionate about. We have worked with Node on multiple hackathon projects and want to explore it further. 
+
+Even before the project list was announced, we were perusing all the then ‘pending’ projects and that was when we found ‘Probot’ and were blown away by its idea almost immediately. We were completely excited by the concept of building Github Apps to automate github workflow. The fact that even we, who are mere beginners here,  could install these apps on our repos and see the magic happen, was enough to transport us to the Probot slack channel within minutes after the final projects announcement on 1st February.
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-$we-init-introductory-post.md]: ' .. after the final projects announcement ..' sounds bit weird. May be change it to '.. after the announcement of final projects.. '. :)

File: blog/_posts/2018-07-09-$we-init-introductory-post.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+* She has motivational one-liners all around her working area- on the white board on her wall, the sticky notes on her desk, on her laptop skin and in her own highly-persistent RAM (yes she remembers a lot from everything inconsequential and forgets the significant). 
+* She loves music, sings a lot but never believes anyone when they tell her she sings well.
+* She believes "Life is all about rising from the ashes you once witnessed yourself reduce to" and possesses a high resilience factor.
+
+## Akshita
+* She is a big foodie, and you will mostly find her in college canteen, eating (xD).
+* She is a Quora lover and loves to draw inspiration from others' lives and stories.
+* She loves to hangout with her friends, in and around Dwarka and visiting the same places again and again. ( She has visited Hashtag and Pacific Mall, innumerable times with her friends).
+* She loves to make a strict time-table of tasks in hand but rarely follows it.
+* Completely opposite to her partner Rahmeen who has an amazing memeory, she is quite forgetful and would be found digging up stuffs.
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-$we-init-introductory-post.md]: Fix the typo here: 'memory'.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Comment by Rahmeen14** [comment]
Hey @nerdyvaishali ! I have made the said changes. Please have a look whenever you find time! Thanks :)

**Review by v-thakkar — APPROVED** [review]

File: blog/_posts/2018-07-09-$we-init-introductory-post.md (hunk 1 above)
Comment (v-thakkar) [reply blog/_posts/2018-07-09-$we-init-introductory-post.md]: Hi @Rahmeen14 ! Permalink should be the same as the file name. Please check other files for the example. Everything else looks good to me.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Review by v-thakkar — APPROVED** [review]

**Comment by v-thakkar** [comment]
Nevermind @Rahmeen14, I've changed it for now as we wanted to publish this post by today. But make sure to follow the blog post guide from the next time. Thanks. :)

**Comment by Rahmeen14** [comment]
Acknowledged! Will take care from next time.

**Pull request merged**